lib/setupmanagers/qemuhck/qemuhck.rb

```diff
@@ -83,6 +83,10 @@ def boot_device
       { 'boot_device' => @project.options.test.boot_device }
     end
 
+    def share_on_host_path
+      { 'share_on_host_path' => @project.options.common.share_on_host_path }
+    end
+
```

Comment:
Why not just put it into the Hash literals in `studio_vm_options` and `client_vm_common_options` like other values?